lib/resque/scheduler/locking.rb

```diff
@@ -97,7 +104,7 @@ def master_lock_key
       end
 
       def redis_master_version
-        Resque.data_store.redis.info['redis_version'].to_f
+        Gem::Version.new(Resque.data_store.redis.info['redis_version'])
```

Comment:
Why the change to using `Gem::Version` here?

Reply:
@iloveitaly this offers "native" version number parsing/comparison which was needed to see when the Redis feature was available.  Specifically, the patch level comparison.